pages/community/dei-mission-statement.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: DEI Mission Statement
+subtitle: Diversity, Equality, and Inclusion at US-RSE
+bigimg:
+permalink: /dei-mission/
+---
+
+We believe Diversity, Equity, and Inclusion (DEI) are core values of the
+US-RSE Association. A dedicated team of active members continue to lead the
+development of a DEI program for our organization. DEI discussions are and
+will continue to be an integral part of our community.
+
+The US-RSE Association is committed to providing an [inclusive environment with
+equitable treatment]({{site.url}}/code-of-conduct) for all and to promoting and
+encouraging diversity throughout the RSE community in the US. At US-RSE,
+integrating DEI practices into our education programs, governance structure,
+and culture is at the forefront of our mission to ensure a welcoming,
+nurturing, and robustly inclusive community. We believe that the
+amplification of diverse perspectives is essential for driving innovation,
+promoting creativity, and encouraging engagement for the success of RSEs. We
+welcome and respect individuals of any race, color, caste, economic status,
+gender, gender identity, sexual orientation, disabilities, age, religion,
+national origin, and ethnicity.
+ 
+The US-RSE Association will:
+- Continually educate ourself through research on best practices focused on
```

Comment:
This sounds more appropriate because there are many of us.
```suggestion
- Continually educate ourslves through research on best practices focused on
```

Reply:
```suggestion
- Continually educate ourselves through research on best practices focused on
```

Reply:
If I recall correctly, we specifically had "ourselves" previously and it was commented in the Google Doc to be replaced with "ourself". Although maybe that was prior to some other change. I agree "ourselves" sounds more grammatically correct.